pg-index-health/src/test/java/io/github/mfvanek/pg/support/statements/AbstractDbStatement.java

```diff
@@ -43,4 +42,18 @@ protected void throwExceptionIfTableDoesNotExist(@Nonnull final Statement statem
             throw new IllegalStateException(String.format("Table with name '%s' in schema '%s' wasn't created", tableName, schemaName));
         }
     }
+
+    protected abstract List<String> getSqlToExecute(@Nonnull String schemaName);
+
+    @Override
+    public void execute(@Nonnull final Statement statement, @Nonnull final String schemaName) throws SQLException {
+        postExecute(statement, schemaName);
+        for (final String sql : getSqlToExecute(schemaName)) {
+            statement.execute(sql);
+        }
+    }
+
+    protected void postExecute(@Nonnull final Statement statement, @Nonnull final String schemaName) throws SQLException {
+        Logger.getLogger(schemaName);
```

Comment:
Remove this line. Just try to add a comment

Reply:
How about using @SuppressFBWarnings and @SuppressWarnings?
Errors do not disappear with simple comments.

Reply:
> How about using @SuppressFBWarnings and @SuppressWarnings? Errors do not disappear with simple comments.

We don't dirty the code with unnecessary annotations. We use an external config. Look at this config/spotbugs/exclude.xml